src/helpers/utils.ts

```diff
@@ -121,6 +121,7 @@ export function calcFromSeconds(value, unit) {
 }
 
 export function calcToSeconds(value, unit) {
+  if (unit === 'm') return value * 60;
```

Comment:
Good you're not married to `calcFromSeconds`. You'd be in trouble now.

Reply:
Right lol, added it there too